### ISSUE TITLE
Sign up form needs to be contained by a <main> element

### ISSUE THREAD

**Issue by emmastephenson** [CDCgov/prime-simplereport#4012]
## Area or Component
Sign up (simplereport.gov/app/sign-up)

## Quick Summary
There's no `<main>` element on the sign up form pages.

### How to Replicate
Visit simplereport.gov/app/sign-up, inspect the page, and see there's no `<main>` component.

## Accessibility Violation
Document should have one main landmark: https://dequeuniversity.com/rules/axe/4.4/landmark-one-main?application=AxeChrome

## Action Requested
Add a `<main>` landmark to all pages of the signup experience (selector, organization form, personal info form, and Experian pages)

## Note
Many of the issues in the signup audit will be fixed by adding a `<main>` landmark, as each sub-element (the org name input, the state input, etc) was also raised for not being contained in a landmark appropriately.

**Comment by emmastephenson** [comment]
Fixed by #4276
